apis/api-media/src/workers/bigQuery/importers/shortLinks/shortLinks.spec.ts

```
@@ -0,0 +1,252 @@
+import { Service as PrismaService } from '.prisma/api-media-client'
```

ESLint

[error] Useless path segments for ".prisma/api-media-client", should be "./.prisma/api-media-client"

(import/no-useless-path-segments)